### PR TITLE
Add `bin/test` script to rails plugin.

### PR DESCRIPTION
**Pull request by y-yagi**
rails minitest extension has become to use the `bin/rails`,  but `bin/rails` does not exist in rails plugin.
Therefore, it is displayed a snippet for rerun, that can not run,
I think it is better that had been disabled by default.

### CODE REVIEW

**Comment by kaspth** [comment]
Our test reporter can be assigned another `executable` to print (which by default is `bin/rails`).

Is there another executable we can assign? I really don't like the approach taken here.

**Comment by arthurnn** [comment]
I am in the same page as @kaspth , I think we should try to make the engines work with it instead of removing it

**Comment by y-yagi** [comment]
I got it. I will fix to work.

**Comment by senny** [comment]
@kaspth I think we should make the runner available from every Rails component, be it a full fledged engine or just a basic railtie plugin.

**Comment by kaspth** [comment]
@senn yup, that would be neat. Can you think of any ways to do that? I can't off the top of my head :)

**Comment by senny** [comment]
Is there anything that prevents us from generating an executable similar to
the bin/test we use in rails/rails?

**Comment by kaspth** [comment]
Not that I can see. It would be nice if we could preserve the `bin/rails test` API.

**Comment by y-yagi** [comment]
`bin/test` looks good.
I add the `bin/test` in plugin, has been updated to use it.

File: railties/test/plugin_test_runner_test.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      RUBY
+    end
+
+    def plugin_file(path, contents, mode: 'w')
+      FileUtils.mkdir_p File.dirname("#{plugin_path}/#{path}")
+      File.open("#{plugin_path}/#{path}", mode) do |f|
+        f.puts contents
+      end
+    end
+end
```
Comment (kaspth) [inline railties/test/plugin_test_runner_test.rb]: We should move this file from `railties/test/plugin_test_runner_test.rb` to `railties/test/generators/plugin_test_runner_test.rb`

**Comment by kaspth** [comment]
One niggle, but otherwise it's good. Thanks and ping me when you've updated it :heart:

(Remember to squash commits and rebase on master if needed :grin:)

**Comment by y-yagi** [comment]
@kaspth Thanks for reviewing! I updated!

**Comment by arthurnn** [comment]
fyi, I edited the title of the PR, for history proposes.

**Comment by kaspth** [comment]
Good idea, since it took such a sharp turn.

**Comment by y-yagi** [comment]
Thanks!

**Pull request merged**